core/sds-aspect-model-validator/src/main/java/io/openmanufacturing/sds/aspectmodel/shacl/violation/ModelFixer.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for additional
+ * information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package io.openmanufacturing.sds.aspectmodel.shacl.violation;
+
+import java.util.List;
+import java.util.function.BiFunction;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+
+import io.openmanufacturing.sds.aspectmodel.shacl.fix.Fix;
+import io.openmanufacturing.sds.aspectmodel.shacl.fix.ReplaceValue;
+
+public class ModelFixer implements BiFunction<Model, List<Fix>, Model> {
```

Comment:
I am sure I am missing something, but as far as I can tell, this class is never used.

Reply:
This is correct, this is a class that is intended to be used by consumers that want to apply a `List<Fix>` to a given model. As long as only the Fix API is place and most violations have no actual `fixes()` implementation, I'd wait with documenting this class in the developer documentation.